src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -211,4 +211,53 @@ public void testDoPost_CleansHtmlContent() throws IOException, ServletException
 
     Mockito.verify(mockResponse).sendRedirect("/chat/test_conversation");
   }
+
+  @Test
+  public void testDoPost_BoldStyling() throws IOException, ServletException {
+      setupInputSanitationTest("[b]Contains[/b] bold [b]styled[/b] content[/b].");
+
+      chatServlet.doPost(mockRequest, mockResponse);
+
+      verifyProperSanitation("<b>Contains</b> bold <b>styled</b> content[/b].");
+  }
+
+  @Test
+  public void testDoPost_ItalicsStyling() throws IOException, ServletException {
+      setupInputSanitationTest("[i]Contains[/i] italics [i]styled[/i] content[/b].");
+
+      chatServlet.doPost(mockRequest, mockResponse);
+
+      verifyProperSanitation("<i>Contains</i> italics <i>styled</i> content[/b].");
+  }
+
+  private void setupInputSanitationTest(String userMessage) {
```

Comment:
So there are tests that are run that have the @Test annotation. However, these are just helper methods. I use them similar to how "setUp" is used, but only for a subset of tests (testDoPost_BoldStyling, testDoPost_ItalicsStyling). I am using #setupInputSanitationTest to setup the test cases, and then #verifyProperSanitation to verify that the tests work (this is instead of duplicating this code)